.github/workflows/ci.yml

```diff
@@ -133,31 +137,75 @@ jobs:
         sudo ./ci/setup_ci_environment.sh
         sudo ./ci/install_gcc48.sh
     - name: setup cmake
-      run: |
-        sudo CC=/usr/bin/gcc-4.8 CXX=/usr/bin/g++-4.8 ./ci/setup_cmake.sh
+      env:
+        CC: /usr/bin/gcc-4.8
+        CXX: /usr/bin/g++-4.8
+        GOOGLETEST_VERSION: "1.10.0"
+      run: sudo -E ./ci/setup_cmake.sh
     - name: setup grpc
-      run: |
-        sudo CC=/usr/bin/gcc-4.8 CXX=/usr/bin/g++-4.8 ./ci/setup_grpc.sh -v 4.8
+      env:
+        CC: /usr/bin/gcc-4.8
+        CXX: /usr/bin/g++-4.8
+        GOOGLETEST_VERSION: "1.10.0"
+      run: sudo -E ./ci/setup_grpc.sh -v 4.8
     - name: run tests
-      run: ./ci/do_ci.sh cmake.legacy.exporter.otprotocol.test
       env:
         CC: /usr/bin/gcc-4.8
         CXX: /usr/bin/g++-4.8
+        GOOGLETEST_VERSION: "1.10.0"
+      run: ./ci/do_ci.sh cmake.legacy.exporter.otprotocol.test
 
-  cmake_test_cxx20:
-    name: CMake C++20 test
+  cmake_test_cxx20_gcc:
+    name: CMake C++20 test(GCC)
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
       with:
         submodules: 'recursive'
     - name: setup
+      env:
+        CMAKE_VERSION: "3.20.6"
       run: |
-        sudo ./ci/setup_ci_environment.sh
-        sudo ./ci/setup_cmake.sh
+        sudo -E ./ci/setup_ci_environment.sh
+        sudo -E ./ci/setup_cmake.sh
     - name: run tests
+      env:
+        CMAKE_VERSION: "3.20.6"
```

Comment:
nit - do we need to set this env variable before the tests?

Reply:
Thanks, it's removed now.